css/src/tokens/themes.scss

```diff
@@ -212,14 +212,14 @@ $themes: (
 		'info-active': $palette-purple-30,
 		'info-box-shadow': $palette-purple-opacity-70,
 		'info-invert': $palette-black,
-		'warning-base': $palette-yellow-50,
-		'warning-background': $palette-yellow-80,
-		'warning-background-hover': $palette-yellow-50,
-		'warning-background-glow-high-contrast': $palette-yellow-80,
-		'warning-dark': $palette-yellow-10,
-		'warning-dark-hover': $palette-yellow-30,
-		'warning-hover': $palette-yellow-40,
-		'warning-active': $palette-yellow-30,
+		'warning-base': $palette-yellow-50-deprecated,
```

Comment:
why there is yellow color used for dark theme? Have you checked this mapping?

https://master--628d031b55e942004ac95df1.chromatic.com/?path=/docs/theme-colors--page